brd-ios/breadwallet/src/Redesign/Demo/DemoModels.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  DemoModels.swift
+//  breadwallet
```

Comment:
no more breadwallet

Reply:
wdym? project name is still breadwallet :o